GettingStarted.md

```diff
@@ -0,0 +1,103 @@
+
+Back at the home project on OBS, you will require target repositories to build against.
+These can be added using a UI but it is easier to add them manually.
+
+1. Click "Advanced" and "Meta"<br>
+   Note, dont confuse this with the package Meta configuration, the nanes are the same
+2. You will be presented with an XML description of the target repositories.  Between
+   the <project> </project> tags, add the following:
+        <repository name="sailfish_latest_i486">
+            <path project="sailfishos:latest" repository="latest_i486"/>
+            <arch>i586</arch>
+        </repository>
+        <repository name="sailfish_latest_armv7hl">
+            <path project="sailfishos:latest" repository="latest_armv7hl"/>
+            <arch>armv8el</arch>
+        </repository>
+        <repository name="sailfish_latest_aarch64">
+            <path project="sailfishos:latest" repository="latest_aarch64"/>
+            <arch>aarch64</arch>
+        </repository>
+        <repository name="sailfish_4.1.0.24_armv7hl">
+            <path project="sailfishos:4.1.0.24" repository="latest_armv7hl"/>
+            <arch>armv8el</arch>
+        </repository>
+        <repository name="sailfish_4.1.0.24_aarch64">
+            <path project="sailfishos:4.1.0.24" repository="latest_aarch64"/>
+            <arch>aarch64</arch>
+        </repository>
+        <repository name="sailfish_4.1.0.24_i486">
+            <path project="sailfishos:4.1.0.24" repository="latest_i408"/>
+            <arch>i586</arch>
+        </repository>`
+   This will create build targets for the 4.1 and latest SaifishOS releases for
+   all supported archutectures.
+3. Click "Save Changes", navigate back to the project overview, and you should see
+   the build status for each target on the right hand side.
+4. You can click on the package name, and then click on the build target status to
+   see the current build log.
+  
+## Submitting your package to saailfishos:chum:testing
+
+Once you are happyy your builds are working, you can submit it for inclusion to chum:testing.
+
+1. Naviate to the package, and click "Submit Package"
+2. Enter the target project as `sailfishos:chum:testing`
+3. Click Ok
+
+For packages submitted to chum, it is probably best to specify a particular git revision to be built.
+For this reason, you may want to set up your home project with builds for your master
+branch for personal use, and a specific revision to be submitted to chum.
+
+## Other useful notes
+
+* You can create sub-projects in your home project to group builds together
+* You can add the repository for the builds to your device using `zypper ar` to install
+  on your device
+* To re-trigger OBS to download the latest code, click the "Trigger Service" link in the package
+* You can configure OBS to not build certain packages against certain repositories, or disable
+  certain projects from being published.  It is very flexible.  This is available from the "Repsitories"
+  link.
```

Comment:
Would it be worth adding something about dependencies (i.e. if I want my project built against libraries in the chum repos)?